ci-operator/config/openshift/lightspeed-operator/openshift-lightspeed-operator-main.yaml

```diff
@@ -0,0 +1,39 @@
+binary_build_commands: make build --warn-undefined-variables
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.21
+images:
+- dockerfile_path: Dockerfile
+  to: openshift-lightspeed-operator
+promotion:
+  to:
+  - name: "4.15"
+    namespace: lightspeed-operator
+releases:
+  initial:
+    integration:
+      name: "4.15"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
```

Comment:
Just a question: According to the [reference](https://steps.ci.openshift.org/ci-operator-reference#line1264),  it "include images built during the test itself", are we including the contents of the test images defined in 
"tests=>container=>from" (line 35) ?🤔

Reply:
yeah i don't think we need or want this, @vbelouso can you explain the intent?